### PR TITLE
Start clarifying pieces of app, adding in navbars

### PR DESCRIPTION
**Pull request by kevinrobinson**
This adds in the `NavigationAppBar` to most screens, and adds some coloring to show which part of the app it is. 

The "assessment machinery" pieces are blue-gray, while Message PopUp is light blue.
screen shot 2016-07-18 at 11 03 27 am

screen shot 2016-07-18 at 11 03 38 am

### CODE REVIEW

**Comment by kevinrobinson** [comment]
@david-rosales Can you review when you get a chance?

File: ui/src/components/navigation_app_bar.jsx [hunk 1]
```diff
@@ -46,6 +47,8 @@ export default React.createClass({
   render() {
     const demoCandidateEmail = 'kevin.robinson.0@gmail.com';
     const {userProfile, doLogout} = this.context.auth;
+    const {iconElementRight} = this.props;
```
Comment (kevinrobinson) [inline ui/src/components/navigation_app_bar.jsx]: This was to allow callers to set the right-side menu based on whatever the application context is (initially for use within a message popup game) but this isn't used yet.

File: ui/src/message_popup/experience_page.jsx [hunk 2]
```diff
@@ -128,17 +128,7 @@ export default React.createClass({
...
-          title='Message PopUp Practice'
-          iconElementLeft=
-          {
-            <IconButton
-              onTouchTap={this.resetExperience}
-              >
-              {!hasStarted ? <MenuIcon/> : questionsAnswered >= sessionLength ? <RefreshIcon /> : <ArrowBackIcon />}
-            </IconButton>
-          }
-          />
```
Comment (kevinrobinson) [inline ui/src/message_popup/experience_page.jsx]: @david-rosales This didn't translate as well with the navigation menu on the left (e.g, having the back button on the right side menu).  I removed it to simplify for now, but if you have any ideas to improve on this that'd be great too!

Comment (david-rosales) [inline ui/src/message_popup/experience_page.jsx]: @kevinrobinson Maybe we can add an experience specific section to the Drawer and in there have the options to go back/restart the message popup game?

**Comment by david-rosales** [comment]
@kevinrobinson  The original `.../home/navigation_app_bar.jsx` is still being used as an import in `.../do_something/do_something_page.jsx` instead of the new `.../components/navigation_app_bar.jsx`. Other than that, it looks good.

**Comment by kevinrobinson** [comment]
@david-rosales ah thanks!  👍   I should have run the tests locally first, fixed up now.

File: ui/src/message_popup/experience_page.jsx (hunk 2 above)
Comment (kevinrobinson) [inline ui/src/message_popup/experience_page.jsx]: Yeah that's an interesting idea!  I'll try that out.

**Comment by kevinrobinson** [comment]
@david-rosales I added this in like you suggested, thanks!

screen shot 2016-07-18 at 5 20 07 pm

**Pull request merged**